### ISSUE TITLE
Bug: Biome parse error with Svelte {#each} blocks in page/layout

### ISSUE THREAD

**Issue by bennajah** [biomejs/biome#8504]
### Environment information

```block
CLI:
  Version:                      2.3.10
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           windows

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         unset
  JS_RUNTIME_VERSION:           v24.12.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         bun/1.3.5

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true
  HTML full support enabled:    true

Workspace:
  Open Documents:               0
```

### What happened?

File: +page.svelte or +layout.svelte

```svelte 
<script lang="ts">
  const locales = ["en", "de"];
</script>

<ul>
  {#each locales as locale}
    <li>{locale}</li>
  {/each}
</ul>
```

<img width="466" height="186" alt="Image" src="https://github.com/user-attachments/assets/4ae1f361-97f9-4ef5-89b5-c51579d406b8" />

### Expected result

- Biome should ignore or correctly parse Svelte template syntax
- No biome(parse) errors for valid {#each} blocks

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
The implementation for this was literally just merged today: #8493 

I went ahead and triggered a [preview release](https://github.com/biomejs/biome/actions/runs/20350422198) if you'd like to test it out. It'll be in the releases channel in our discord when its done building.
